pallets/msa/src/mock.rs

```diff
@@ -180,7 +178,17 @@ impl pallet_msa::Config for Test {
 	>;
 }
 
+pub fn set_max_signature_stored(max: u32) {
+	MAX_SIGNATURES_STORED.with(|v| *v.borrow_mut() = Some(max));
+}
+
+pub fn set_max_public_keys_per_msa(max: u8) {
+	MAX_PUBLIC_KEYS_PER_MSA.with(|v| *v.borrow_mut() = max);
+}
+
 pub fn new_test_ext() -> sp_io::TestExternalities {
+	set_max_signature_stored(8000);
```

Comment:
This is a nice improvement. I recommend using the builder pattern instead of having multiple new_test_ext(); I can see this being a little confusing when we are adding a test. Not a blocker.

Perhaps, we can add another ticket to continue with the refactor.

```
ExtBuilder::default()
  . set_max_sigature_store(..)
  . set_max_public_keys_per_msa()
  .build()
  .execute_build_with(|| {
    ....assert
  })
  ```

Reply:
yes.  that is coming in another story/pr.  in the works!

Reply:
https://github.com/LibertyDSNP/frequency/issues/1210